dagbot/bot.py

```diff
@@ -28,62 +28,67 @@ async def get_prefix(bot, message):
 
 
 class Dagbot(commands.AutoShardedBot):
+
     def __init__(self):
         super().__init__(
             command_prefix=get_prefix,
             description='The number 1 wanna be meme bot',
             case_insensitive=True,
             allowed_mentions=discord.AllowedMentions(
                 roles=False,
-                everyone=False))
+                everyone=False)
+        )
+
         with open('./dagbot/data/credentials.yml', 'r') as file:
             self.data = yaml.load(file, Loader=yaml.FullLoader)
+
+        self.launch_time = None
+        self.session = None
+        self.pg_con = None
+
         self.caching = caching(self)
         self.dagpi = Client(self.data['dagpitoken'])
         self.bwordchecker = bword()
         self.bwordchecker.loadbword()
         self.useage = {}
         self.commands_called = 0
+
         # self.add_cog(Help(bot))
+
         self.load_extension("jishaku")
+
         extensions = [
-            "text",
-            "fun",
-            "newimag",
-            "reddit",
-            "games",
-            "util",
-            "whysomart",
-            "animals",
-            "memes",
-            "tags",
-            "misc",
-            "settings",
-            "ai",
-            "events",
-            "errors",
-            "developer",
-            "help"
+            "text", "fun", "newimag",
+            "reddit", "games", "util",
+            "whysomart", "animals", "memes",
+            "tags", "misc", "settings",
+            "ai", "events", "errors",
+            "developer", "help"
```

Comment:
I don't think black with agree with these

Reply:
Black isn't the end all be all of code style.

Reply:
I agree, I personally plan on moving away from black. I will be doing some more formatting on my own!

Reply:
As far as I know, repo author uses it as primary formatter

Reply:
Alright